Makefile

```diff
@@ -3,15 +3,22 @@ export
 SHELL := /bin/bash
 LOCAL := $(PWD)/usr
 PATH := $(LOCAL)/bin:$(PATH)
-TESSDATA =  $(LOCAL)/share/tessdata 
-LANGDATA = $(PWD)/langdata-$(LANGDATA_VERSION)
+HOME := /home/ubuntu
```

Comment:
`$HOME` is set by default (i.e. `HOME := $(HOME)`), if you want to override it you can, but I would not set a different default. As you use it, it seems more like you meant `PWD` or `REPO_ROOT`?